xtask/src/cli.rs

```diff
@@ -125,22 +127,23 @@ pub fn run(command: &mut Command) -> Result<()> {
         println!("Running: {}", command_str);
     }
 
-    let status = command.spawn()?.wait()?;
+    let output = command.output()?;
```

Comment:
Hmmm I think this change isn't right. This changes things to swallow all the output from lots of `xtask` commands, e.g. the Python examples https://github.com/PyO3/pyo3/runs/6846083356?check_suite_focus=true#step:20:97

Maybe there needs to be two variants of `run`? `run` and `run_output` ?

Reply:
Oops, thanks. I've created it.